packages/near-fast-auth-signer/src/utils/multi-chain/chains/EVM.ts

```diff
@@ -106,17 +109,19 @@ class EVM {
    * @returns {Promise<ethers.providers.TransactionRequest>} A new transaction object augmented with gas price, gas limit, chain ID, and nonce.
    */
   async attachGasAndNonce(
-    transaction: ethers.TransactionLike & {
-      from: string;
-    }
+    transaction: EVMTransaction & { from: string }
```

Comment:
The difference is that in our input we allow the user to provide us with a nice eth string and convert to Wei bigint for them instead of requiring them to pass that